Adyen/Utilities/Localization.swift

```diff
@@ -41,7 +42,16 @@ public func localizedString(_ key: LocalizationKey, _ parameters: LocalizationPa
 }
 
 private func fallbackLocalizedString(key: String) -> String {
-    NSLocalizedString(key, tableName: nil, bundle: Bundle.coreInternalResources, comment: "")
+    let localizedFallback = NSLocalizedString(key, tableName: nil, bundle: Bundle.coreInternalResources, comment: "")
+    
+    if localizedFallback != key && localizedFallback.isEmpty == false {
+        return localizedFallback
+    } else {
+    // Fallback to en-US
+        return Bundle.coreInternalResources.path(forResource: "en-US", ofType: "lproj")
+            .flatMap(Bundle.init(path: ))
+            .map { NSLocalizedString(key, tableName: nil, bundle: $0, comment: "") } ?? key
```

Comment:
why not put this in the `buildPossibleInputs`?

Reply:
this is more of a fallback option, so i thought it should be in `fallbackLocalizedString`

Reply:
yeah, actually you're right, here is better.

but can we unit test this in `ADYLocalizationTests`?

Reply:
unfortunately i have not found a reliable way to test this

Reply:
👍 no problem.